examples/go-client/terraform/resources/README.md

```diff
@@ -10,7 +21,7 @@ Name | Type | Requirement | Description
 **path_account_sid** | string | Optional | 
 **recording_status_callback** | string | Optional | 
 **recording_status_callback_event** | list(string) | Optional | 
-**sid** | int | *Computed* | INTEGER SID!!!
+**test_integer** | int | *Computed* | INTEGER ID param!!!
```

Comment:
If I'm understanding your approach correctly, the `test_blahs` are meant to be a catch-all params that remove the burden to autogenerate specific parameters (i.e. SIDs, urls, etc), and `TestResponseObject` accomplishes the same thing except for API responses?

Reply:
Yeah, basically. Just a generic way of validating we don't break the generator when a specific type is used.